pdf/pdfcanvas.cpp

```diff
@@ -78,6 +89,12 @@ class PDFCanvas::Private
     float spacing;
 
     QRectF visibleArea;
+    bool rememberPagePosition;
+    QString dSource;
```

Comment:
Thinking if it would be cleaner for this reason too, if instead of dtor the saving would be explicitly asked on inactive application or popped page.